internal/generate/crd/crd_test.go

```diff
@@ -300,14 +300,10 @@ spec:
       description: Memcached is the Schema for the memcacheds API
       properties:
         apiVersion:
-          description: 'APIVersion defines the versioned schema of this representation
-            of an object. Servers should convert recognized schemas to the latest
-            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+          description: 'APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
           type: string
         kind:
-          description: 'Kind is a string value representing the REST resource this
-            object represents. Servers may infer this from the endpoint the client
-            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+          description: 'Kind is a string value representing the REST resource this object represents. Servers may infer this from the endpoint the client submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
```

Comment:
The newlines present in-between strings were not passing the tests. Had to remove them.

Reply:
They pass in master.

Reply:
What was the error @varshaprasad96?

Reply:
There were errors due to the present of newline in the expected string output with backticks (back quotes). This was after bumping versions of some of the dependencies along with controller-runtime (though I didn't look into what was the exact one causing this). This does not happen in master. For ex in case of one of the test `TestCRDGo`:

The expected was:
```expected: "apiVersion: apiextensions.k8s.io/v1beta1\nkind: CustomResourceDefinition\nmetadata:\n  name: memcacheds.cache.example.com\nspec:\n  group: cache.example.com\n  names:\n    kind: Memcached\n    listKind: MemcachedList\n    plural: memcacheds\n    singular: memcached\n  scope: Namespaced\n  subresources:\n    status: {}\n  validation:\n    openAPIV3Schema:\n      description: Memcached is the Schema for the memcacheds API\n      properties:\n        apiVersion:\n\t\t  description: 'APIVersion defines the versioned schema of this representation \n\t\t  of an object. Servers should convert recognized schemas to the latest \n\t\t  internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'\n          type: string\n        kind:\n\t\t  description: 'Kind is a string value representing the REST resource this \n\t\t  object represents. Servers may infer this from the endpoint the client \n\t\t  submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'\n          type: string\n        metadata:\n          type: object\n        spec:\n          description: MemcachedSpec defines the desired state of Memcached\n          properties:\n            size:\n              description: Size is the size of the memcached deployment\n              format: int32\n              type: integer\n          required:\n          - size\n          type: object\n        status:\n          description: MemcachedStatus defines the observed state of Memcached\n          properties:\n            nodes:\n              description: Nodes are the names of the memcached pods\n              items:\n                type: string\n              type: array\n          required:\n          - nodes\n          type: object\n      type: object\n  version: v1alpha1\n  versions:\n  - name: v1alpha1\n    served: true\n    storage: true\n"```

And the actual is:
```
"apiVersion: apiextensions.k8s.io/v1beta1\nkind: CustomResourceDefinition\nmetadata:\n  name: memcacheds.cache.example.com\nspec:\n  group: cache.example.com\n  names:\n    kind: Memcached\n    listKind: MemcachedList\n    plural: memcacheds\n    singular: memcached\n  scope: Namespaced\n  subresources:\n    status: {}\n  validation:\n    openAPIV3Schema:\n      description: Memcached is the Schema for the memcacheds API\n      properties:\n        apiVersion:\n          description: 'APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'\n          type: string\n        kind:\n          description: 'Kind is a string value representing the REST resource this object represents. Servers may infer this from the endpoint the client submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'\n          type: string\n        metadata:\n          type: object\n        spec:\n          description: MemcachedSpec defines the desired state of Memcached\n          properties:\n            size:\n              description: Size is the size of the memcached deployment\n              format: int32\n              type: integer\n          required:\n          - size\n          type: object\n        status:\n          description: MemcachedStatus defines the observed state of Memcached\n          properties:\n            nodes:\n              description: Nodes are the names of the memcached pods\n              items:\n                type: string\n              type: array\n          required:\n          - nodes\n          type: object\n      type: object\n  version: v1alpha1\n  versions:\n  - name: v1alpha1\n    served: true\n    storage: true\n"
```

Reply:
@varshaprasad96 I saw the same error in my new PR #3523. I think the new version of ginkgo or gomega caused it.

Reply:
Oh..The failing tests here are written in the old golang testing framework though. Not sure which, but some other dependency maybe causing it.